addons/website/static/src/builder/plugins/options/customers_avatars_option_plugin.js

```diff
@@ -0,0 +1,97 @@
+import { Plugin } from "@html_editor/plugin";
+import { registry } from "@web/core/registry";
+import { BuilderAction } from "@html_builder/core/builder_action";
+class AvatarCountAction extends BuilderAction {
+    static id = "avatarCount";
+
+    getValue({ editingElement, params: { mainParam: action } = {} }) {
+        if (action === 'countValue') {
+            return editingElement.dataset.countValue || '10+';
+        }
+        if (action === 'showCount') {
+            return editingElement.classList.contains('o_show_count');
+        }
+        return null;
+    }
+
+    isApplied({ editingElement, params: { mainParam: action } = {} }) {
+        if (action === 'showCount') {
+            const countElement = editingElement.querySelector('.o_avatar_count');
+            return countElement && !countElement.classList.contains('d-none');
+        }
+        return false;
+    }
+
+    apply({ editingElement, params: { mainParam: action } = {}, value }) {
+        if (action === 'showCount') {
+            const countElement = this._ensureCountElement(editingElement);
+            if (countElement) {
+                countElement.classList.toggle('d-none');
+                countElement.classList.toggle('d-flex');
+                this._copyStyling(editingElement);
+            }
+        } else if (action === 'countValue') {
+            editingElement.dataset.countValue = value || '10+';
+            const countElement = editingElement.querySelector('.o_avatar_count');
+            if (countElement) {
+                countElement.textContent = value || '10+';
+            }
+        } else if (action === 'copyStyling') {
+            this._copyStyling(editingElement);
+        }
+    }
+
+    _ensureCountElement(editingElement) {
+        let countElement = editingElement.querySelector('.o_avatar_count');
+        if (!countElement) {
+            const wrapper = editingElement.querySelector('.s_customers_avatars_wrapper');
+            if (wrapper) {
+                countElement = document.createElement('div');
+                countElement.className = 'o_avatar_count d-none';
+                countElement.textContent = editingElement.dataset.countValue || '10+';
+                wrapper.appendChild(countElement);
+            }
+        }
+        return countElement;
+    }
+
+    _copyStyling(editingElement) {
+        const countElement = editingElement.querySelector('.o_avatar_count');
+        const firstAvatar = editingElement.querySelector('.o_avatar');
+
+        if (countElement && firstAvatar) {
+            const isHidden = countElement.classList.contains('d-none');
+            const isVisible = countElement.classList.contains('d-flex');
+
+            countElement.className = 'o_avatar_count';
+
+            if (isHidden) countElement.classList.add('d-none');
+            if (isVisible) countElement.classList.add('d-flex');
```

Comment:
- No one liners, we enforce the use of `{`
- You should not have a isHidden and a isVisible, it's one or the other no?
- Use classlist.toggle